client/styles/ui-components/Chakra-Button.tsx

```diff
@@ -1,140 +1,151 @@
 import { Button, ButtonGroup } from '@chakra-ui/react';
+import type { ButtonProps } from '@chakra-ui/react';
```

Comment:
provides all the types for the wrapper component.

Reply:
This is a great addition! THank you!